db/seeds.rb

```diff
@@ -6,3 +6,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
![image](https://user-images.githubusercontent.com/11413458/56655035-08b19b00-6668-11e9-9dd6-0ae8d1864628.png)

This image, says: you need to add an empty line at the end of file. If you are using Visual Studio Code, you can add a this configuration: 
```
"files.insertFinalNewline": true
```

Reply:
Done